astropy/io/misc/tests/test_parquet_votable.py

```diff
@@ -0,0 +1,90 @@
+import numpy as np
+import pytest
+
+import astropy.units as u
+from astropy.io.misc.parquet import read_parquet_votable, write_parquet_votable
+from astropy.table import Table
+from astropy.utils.exceptions import AstropyUserWarning
+from astropy.utils.misc import _NOT_OVERWRITING_MSG_MATCH
+
+# Skip all tests in this file if we cannot import pyarrow
+pyarrow = pytest.importorskip("pyarrow")
+
+
+number_of_objects = 10
+ids = [f"COSMOS_{ii}" for ii in range(number_of_objects)]
+redshift = np.random.uniform(low=0, high=3, size=number_of_objects)
+mass = np.random.uniform(low=1e8, high=1e10, size=number_of_objects)
+sfr = np.random.uniform(low=1, high=100, size=number_of_objects)
+
+input_table = Table(
```

Comment:
It would be interesting to have test cases where the input astropy table was created by reading an actual votable with its own metadata.

Reply:
added a test example for this